com.zsmartsystems.zigbee.dongle.xbee/src/main/java/com/zsmartsystems/zigbee/dongle/xbee/ZigBeeDongleXBee.java

```diff
@@ -337,6 +337,10 @@ public void sendCommand(final int msgTag, final ZigBeeApsFrame apsFrame) {
             command.addOptions(TransmitOptions.ENABLE_APS_ENCRYPTION);
         }
 
+        if (!apsFrame.getAckRequest()) {
+            command.addOptions(TransmitOptions.DISABLE_RETRIES);
```

Comment:
Please, can you test that the frame is indeed transmitted with APS Ack Request = FALSE as I don't have a dongle to test with.